CONTRIBUTING.md

```diff
@@ -319,7 +319,7 @@ nothing to commit, working directory clean
     add .` to add all unstaged files. Take care, though, because you can
     accidentally add files you don't want added. Review your `git status` first.
 
-6.  Commit your edits: `git commit -m "Brief Description of Commit"`
+6.  Commit your edits: `git commit -m "Brief Description of Commit"`. Do not add the issue number in the commit message.
```

Comment:
@raisedadead It would be very confusing if someone reads that instruction way later than where commits are explained. @dhcodes